src/modules/producer.js

```diff
@@ -81,27 +109,28 @@ function publishOrSendToQueue(queue, msg, options) {
  * @param  {object} options contain rpc property (if true, enable rpc for this message)
  * @return {Promise}         Resolves when message is correctly sent, or when response is received when rpc is enabled
  */
-function checkRpc (queue, msg, options) {
+function checkRpc(queue, msg, options) {
   //messages are persistent
   options.persistent = true;
-
+  
   if (options.rpc) {
     return createRpcQueue.call(this, queue)
-    .then(() => {
-      //generates a correlationId (random uuid) so we know which callback to execute on received response
-      var corrId = uuid.v4();
-      options.correlationId = corrId;
-      //reply to us if you receive this message!
-      options.replyTo = amqpRPCQueues[queue].queue;
-
-      publishOrSendToQueue.call(this, queue, msg, options);
-
-      //defered promise that will resolve when response is received
-      amqpRPCQueues[queue][corrId] = Promise.defer();
-      return amqpRPCQueues[queue][corrId].promise;
-    });
+      .then(() => {
+        //generates a correlationId (random uuid) so we know which callback to execute on received response
+        var corrId = uuid.v4();
+        options.correlationId = corrId;
+        //reply to us if you receive this message!
+        options.replyTo = amqpRPCQueues[queue].queue;
+        
+        return publishOrSendToQueue.call(this, queue, msg, options)
+          .then(() => {
+            //defered promise that will resolve when response is received
+            amqpRPCQueues[queue][corrId] = Promise.defer();
+            return amqpRPCQueues[queue][corrId].promise;
+          });
+      });
   }
-
+  
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.dialonce.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)